src/applications/mhv-secure-messaging/tests/e2e/keyboard-nav-test/secure-messaging-compose-error-message-keyboard-nav.cypress.spec.js

```diff
@@ -4,7 +4,7 @@ import PatientComposePage from '../pages/PatientComposePage';
 import FolderLoadPage from '../pages/FolderLoadPage';
 import { AXE_CONTEXT, Data } from '../utils/constants';
 
-describe('Secure Messaging Compose Errors Keyboard Nav', () => {
+describe.skip('Secure Messaging Compose Errors Keyboard Nav', () => {
```

Comment:
as discussed [here](https://dsva.slack.com/archives/C03ECSBGSKX/p1718720331175189) all skipped tests will be fixed after merge